manifests/2.5.0/opensearch-2.5.0.yml

```diff
@@ -108,7 +108,6 @@ components:
     platforms:
       - linux
       - windows
-    working_directory: opensearch-observability
```

Comment:
Have we removed all front end folders in this repo and make it exclusive for backend? If not, removing the `working_directory` may cause build failure.

Reply:
there's a PR ready to merge which removes all front end stuff, will get it merged soon